core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/Debian.yml

```diff
@@ -55,3 +55,57 @@
   include_tasks: "replication-Debian.yml"
   when:
     - specification.replication is defined and specification.replication.enable is defined and specification.replication.enable
+
+- name: Install pgbouncer package
+  apt:
+    name:
+      - pgbouncer
+    update_cache: yes
+    state: present
+  when:
+    - specification.additional_components is defined
+    - specification.additional_components.pgbouncer.enable
```

Comment:
Shouldn't you also add checking if pgbouncer is defined? This could be a problem when additional_components are defined, but there is no configuration for pgbouncer.